docs/source/guide/tutorials/base/how_to_train/anomaly_detection.rst

```diff
@@ -1,4 +1,190 @@
+
+    MultiScorePerformance(score: 0.9032258064516128, primary_metric: None, additional_metrics: (1 metrics), dashboard: (1 metric groups))
+
+******
+Export
+******
+
+1. ``otx export`` exports a trained Pytorch `.pth` model to the OpenVINO™ Intermediate Representation (IR) format.
+It allows running the model on the Intel hardware much more efficient, especially on the CPU. Also, the resulting IR model is required to run POT optimization. IR model consists of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
+
+2. We can run the below command line to export the trained model
+and save the exported model to the ``openvino_models`` folder.
+
+.. code-block::
+
+    otx export ote_anomaly_detection_padim \
+        --load-weights otx-workspace-ANOMALY_DETECTION/models/weights.pth \
+        --save-model-to otx-workspace-ANOMALY_DETECTION/openvino_models
+
+You will see the outputs similar to the following::
+
+    [INFO] 2023-02-21 16:42:43,207 - otx.algorithms.anomaly.tasks.inference - Initializing the task environment.
+    [INFO] 2023-02-21 16:42:43,632 - otx.algorithms.anomaly.tasks.train - Loaded model weights from Task Environment
+    [WARNING] 2023-02-21 16:42:43,639 - otx.algorithms.anomaly.tasks.inference - Ommitting feature dumping is not implemented.The saliency maps and representation vector outputs will be dumped in the exported model.
+    [INFO] 2023-02-21 16:42:43,640 - otx.algorithms.anomaly.tasks.inference - Exporting the OpenVINO model.
+    [ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.
+    Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/latest/openvino_2_0_transition_guide.html
+    [ SUCCESS ] Generated IR version 11 model.
+    [ SUCCESS ] XML file: /tmp/otx-anomaliba3imqkmo/onnx_model.xml
+    [ SUCCESS ] BIN file: /tmp/otx-anomaliba3imqkmo/onnx_model.bin
+
+Now that we have the exported model, let's check its performance using ``otx eval``
+
+.. code-block:: bash
+
+    otx eval ote_anomaly_detection_padim \
+        --test-data-roots data/anomaly/hazelnut_toy \
+        --load-weights otx-workspace-ANOMALY_DETECTION/openvino_models/openvino.xml \
+        --save-performance otx-workspace-ANOMALY_DETECTION/openvino_models/performance.json
+
+This gives the following results::
+
+    MultiScorePerformance(score: 0.8974358974358974, primary_metric: None, additional_metrics: (1 metrics), dashboard: (1 metric groups))
+
+************
+Optimization
+************
+
+Anomaly tasks can be optimized either in POT or NNCF format. For more information refer to the :doc:`optimization explanation <../../../explanation/additional_features/models_optimization>` section.
+
+
+----------------
+POT Optimization
```

Comment:
Maybe don't make them as stand-alone sections? I prefer combine them as for other tutorials

Reply:
I added these as sub-headings to make it more readable. But I don't mind merging these